views/wishCardFullPage.ejs

```diff
@@ -40,44 +44,39 @@
-                        <a class="dropdown-item" href="#">Happy holidays, <%= wishcard.childFirstName %></a>
-                        <a class="dropdown-item" href="#"><%= wishcard.childFirstName %>, you are awesome!</a>
-                        <a class="dropdown-item" href="#">Lots of love and best wishes, <%= wishcard.childFirstName %></a>
-                        <a class="dropdown-item" href="#"><%= wishcard.childFirstName %>, hope you enjoy my gift!</a>
-                        <a class="dropdown-item" href="#">Merry Christmas and a Happy New Year</a>
-                        <a class="dropdown-item" href="#"><%= wishcard.childFirstName %>, have a happy holiday</a>
-                        <a class="dropdown-item" href="#">Congratulations, <%= wishcard.childFirstName %></a>
-                    </div>
-                </div>
-
-                <input type="submit" id="submit" class="mt-4" value="Send Message">
-
+                
+                <form class="message" action="/wishcards/message" method="post">
+                    <select class="custom-select">
+                        <option selected>Messages</option>
+                        <option value="1"><%= user.fName %> sends you love, <%= wishcard.childFirstName %></option>
+                        <option value="2">Happy Birthday to the sweetest kid in the entire world.</option>
+                        <option value="3">Happy birthday to a future superstar!</option>
+                        <option value="4">Happy birthday, <%= wishcard.childFirstName %></option>
+                        <option value="5">Merry Christmas, <%= wishcard.childFirstName %></option>
+                        <option value="6">Happy holidays, <%= wishcard.childFirstName %></option>
+                        <option value="7"><%= wishcard.childFirstName %>, you are awesome!</option>
+                        <option value="8">Lots of love and best wishes, <%= wishcard.childFirstName %></option>
+                        <option value="9"><%= wishcard.childFirstName %>, hope you enjoy my gift!</option>
+                        <option value="10">Merry Christmas and a Happy New Year</option>
+                        <option value="11"><%= wishcard.childFirstName %>, have a happy holiday</option>
+                        <option value="12">Congratulations, <%= wishcard.childFirstName %></option>
+                    </select>
+                    <input type="submit" id="submit" class="mt-4" value="Send Message">
+                </form>
+                
             </div>
 
             <!-- This part needs to show all users' messages sent to this wishcard 
             wishcard model has 'messages' field which is an objectId array of 'Message'-->
-            <div class="show-msg"> 
-                <% if (wishcard.messages.length === 0) { %>
+            <div class="show-msg">
+                <% if (messages.length === 0) { %>
```

Comment:
miising wishcard.?

Reply:
Ah different variables, a bit confusing, whats the difference here between wishcard.messages and messages, maybe a different variable name to clarify?

Reply:
Yeah, probably a good idea to do that. I will change 73 to just messages.

Reply:
ah wait you replaced all wishcard.messages with messages?
sorry git is a bit confusing sometimes

Reply:
Yeah, no worries :+1: